disperser/controller/dispatcher.go

```diff
@@ -174,6 +176,19 @@ func (d *Dispatcher) HandleBatch(ctx context.Context) (chan core.SigningMessage,
 			continue
 		}
 
+		host, v2DispersalPort, err := net.SplitHostPort(operatorSocket.GetV2DispersalSocket())
+		if err != nil {
+			d.logger.Warn("failed to get node client; incorrect operator socket format", "operator", opID.Hex(), "host", host, "v2DispersalPort", v2DispersalPort, "err", err)
```

Comment:
This is an error instead of warning. There may be other spots still using warning in this PR.